src/Context/SoapContext.php

```diff
@@ -102,6 +108,22 @@ public function iRegisterXpathNamespaces(TableNode $namespaces)
     }
 
     /**
+     * @Given /^(?:|I )expect SOAP exception(?:| with code "(\d+)")(?:|( and| or)? with message "([^"]+?)")$/
+     */
+    public function expectException($code = null, $condition = null, $message = null)
+    {
+        // Exit with an error because we're expected an exception and got nothing.
+        if (null === $this->fault) {
+            throw new \RuntimeException('Expected \SoapFault exception was not thrown!');
+        }
+
+        new SoapFaultProcessor($this->fault, $code, $message, $condition);
+
+        // If processor was not thrown an exception then we shouldn't too.
```

Comment:
WIll it sound better if we reword it to `If processor didn't throw an exception, then we shouldn't too`?